### PR TITLE
added table of involved cognate sets to single meaning view

### PR DESCRIPTION
**Pull request by Bibiko**

### CODE REVIEW

File: cobl2/datatables.py [hunk 1]
```diff
@@ -168,22 +184,28 @@ def base_query(self, query):
...
+        if not self.parameter:
+            cols.append(LinkCol(self, 'name', model_col=Meaning.name,
                 get_object=lambda cc: cc.meaning_rel,
-                sTitle='Meaning'),
+                sTitle='Meaning'))
+        return cols + [
             CoblRootFormCol(self, 'Root_form', model_col=CognateClass.root_form),
             CoblRootLanguageCol(self, 'Root_language', model_col=CognateClass.root_language,
-                sTitle='Root ref. language',
+                sTitle='Root re f. language',
```
Comment (xrotwang) [inline cobl2/datatables.py]: Id this a typo?

File: cobl2/datatables.py [hunk 2]
```diff
@@ -481,7 +503,6 @@ def includeme(config):
     # "manually", i.e. in cobl2.main.
     #
     config.register_datatable('values', Forms)
-    config.register_datatable('cognatesets', CognateClasses)
```
Comment (xrotwang) [inline cobl2/datatables.py]: If the table isn't registered anymore, shouldn't the code be removed?

File: cobl2/datatables.py (hunk 1 above)
Comment (Bibiko) [reply cobl2/datatables.py]: oh - good catch - yes - wait

File: cobl2/datatables.py (hunk 2 above)
Comment (Bibiko) [reply cobl2/datatables.py]: it is already registered in cobl2.main as the note suggest

**Pull request merged**